.github/CONTRIBUTING.md

```diff
@@ -58,11 +58,15 @@ This library will load a file from the `config` folder, based on an environment
 ### Setting up the config
 
 1. Create a file named `development.json` at [../config/](../config/).
-2. Copy the contents of [../config/staging.json](../config/staging.json).
-3. In your `.env` file set `NODE_CONFIG_ENV` to `development`.
-4. Change the `guildId` field to your server's ID.
+2. Paste the contents of [../config/staging.json](../config/staging.json) into `development.json`.
+4. In your `development.json` file replace the `guildId` field with your server ID.
 5. Update the channel and role IDs to those in your server.
-6. Add your own user ID and that of your test bot to `modLogsExclude`.
+6. Add your own user ID and that of your test bot to `modLogExclude `.
```

Comment:
```suggestion
6. Add your test bot ID to `modLogExclude ` and `userLogExclude`.
```

I didn't spot this in a previous PR, but lets change it here. It can be very useful to not have your user ID in the excludes, as it replicates production in this way. Then you can test things like scam logging, message edits and deletes. We also need to add the bot to user excludes so that we don't get a 'ping pong' of logs where the bot edits a message (edit message logs for example) then triggers another user log, getting itself in a loop.